src/app/config.rs

```diff
@@ -55,4 +55,24 @@ impl Config {
             reviewed_patchsets_path,
         }
     }
+
+    /// Creates the needed directories if they don't exist.
+    /// The directories are defined during the Config build.
+    ///
+    /// This function must be called as soon as the Config is built so no other function attempt to use an inexistent folder.
+    pub fn create_dirs(&self) {
+        let paths = vec![
+            &self.patchsets_cache_dir,
+            &self.bookmarked_patchsets_path,
+            &self.mailing_lists_path,
+            &self.reviewed_patchsets_path,
```

Comment:
These three paths point to JSON files.

The absence of these three files is "expected" as (when starting `patch-hub`) bookmarked patchsets, reviewed patchsets, and mailing lists are all loaded from the respective JSON and fall back to empty collections in case the file doesn't exist. The creation/update of these is done whenever we need to commit to a new state for these three entities.

Reply:
Oh didn't paid attention to this particular paths. I'll remove them